HHS/OPRE-OPS
# Agreement metadata and budget lines editing outside of wizard (from details page)

## What changed

* agreement metadata editing and budget lines editing has been refactoring into reusable forms
* new agreement details/metadata stand alone editor (from the details page)
* new budget lines stand alone editor (from the details page)
* edit link on the agreements list now goes to the new agreement editor

## Issue

#1209 
#681 
#856

## How to test
* create a test CONTRACT agreement which will verify that the refactored components work in the wizard UI
* on the agreements list, click on the title to go the Agreement details page
* click on the "Edit" link on the right of page
* makes change to the agreement metadata and click "Save Changes" at the bottom
* click on the "Budget Lines" tab
* click on the "Edit" link on the right of page
* edit the budget lines and click "Save Changes" at the bottom
  * NOTE: you cannot delete previously save budget lines yet

## Screenshots

### Screenshot: Agreement Details Editor
![image](https://github.com/HHS/OPRE-OPS/assets/127965976/3c4ef3a2-36e1-4fc4-aecf-a303f0c071bd)

### Screenshot: Budget Lines Editor
![image](https://github.com/HHS/OPRE-OPS/assets/127965976/e6e500db-bc11-462c-8b81-8f45acb078b4)
